docs/source/introduction/guide_custom_nodes.md

```diff
@@ -206,3 +206,27 @@ class CSVLogger:
 **NOTE:**
 
 While running, the csv file may be empty. This is because the implementation of this csv logger completes the writing at the end of the instruction.
+
```

Comment:
Suggest incorporating this in sections 1 and 2 rather than a note at the end as there's a lot of duplicated instructions here. All further references to "custom_nodes" folder in subsequent sections can be replaced by <folder_name>.